components/script/dom/document.rs

```diff
@@ -2145,8 +2141,8 @@ impl Document {
             // for the interim frames where we are deciding whether this rAF
             // is considered spurious, we need to ensure that the layout
             // and compositor *do* tick the animation.
-            self.window
```

Comment:
Looks like the comment here needs updating.

Reply:
I'll update this one.